pkg/csi/service/common/placementengine/placement.go

```diff
@@ -246,6 +246,54 @@ func getExpandedTopologySegments(ctx context.Context, requestedSegments map[stri
 	return completeTopologySegments, nil
 }
 
+// GetAllAccessibleDSInTopology returns a list of accessible datastores for requested topology.
+func GetAllAccessibleDSInTopology(ctx context.Context,
```

Comment:
Can we continue to use getCandidateDSInTopologyForFileVolumes instead? We can modify some parts of it

Reply:
Code is exactly the same. The function was just renamed and moved to `placement.go` file